### PR TITLE
Exams by Rostislav Hundarev | IM-21

### PR DESCRIPTION
**Pull request by RostislavHundarev**
fixed month.js and words.js

### CODE REVIEW

File: Tasks/month.js [hunk 1]
```diff
@@ -1,5 +1,3 @@
-// Get month number
-
 Months = [
```
Comment (darr4i) [inline Tasks/month.js:1]: add const

File: Tasks/month.js [hunk 2]
```diff
@@ -16,7 +14,9 @@ Months = [
 ];
 
 Month = (s) => {
```
Comment (darr4i) [inline Tasks/month.js:16]: add const for arrow function

Comment (RostislavHundarev) [reply Tasks/month.js:16]: Thank you. I add this to my code.

File: Tasks/words.js [hunk 3]
```diff
@@ -3,16 +3,12 @@
 Words = function (s) {
```
Comment (darr4i) [inline Tasks/words.js:3]: rewrite to arrow function

Comment (RostislavHundarev) [reply Tasks/words.js]: Yes, i rewrited this.

**Comment by Kabadzh0b** [comment]
LGTM

File: Tasks/words.js [hunk 4]
```diff
@@ -1,41 +1,18 @@
...
-        } else {
-          flag = true;
-        }
-      }
+const words = (s) => {
+  let numberOfWords = 0;
+  let inWord = false;
+  for (let i = 0; i < s.length; i++) {
+    if (s[i] === ' ') {
+        inWord = false;
```
Comment (tshemsedinov) [inline Tasks/words.js:9]: Can be implemented with: `inWord = !inWord;`

File: Tasks/month.js [hunk 5]
```diff
@@ -15,12 +16,10 @@ Months = [
...
 ];
 
-Month = (s) => {
-  l = Months.length;
-  for (let i = 0; i < l; i++) {
-    if (s.toLowerCase().startsWith(Months[i])) return i + 1;
-  }
-  return -1;
+const month = (s) => {
+  const month = s.toLowerCase().substring(0, 3);
```
Comment (tshemsedinov) [inline Tasks/month.js:20]: Hardcoded magic number: `3`

**Review by tshemsedinov — CHANGES_REQUESTED** [review]
Use eslint before commit

File: Tasks/words.js (hunk 4 above)
Comment (RostislavHundarev) [reply Tasks/words.js:9]: Used ESlint and fixed in my repository: https://github.com/RostislavHundarev/fixed-exams

File: Tasks/month.js (hunk 5 above)
Comment (RostislavHundarev) [reply Tasks/month.js:20]: Used assignment for magic number: 3
https://github.com/RostislavHundarev/fixed-exams

**Pull request closed without merging**